pinpoint-hbase/hbase-env.sh

```diff
@@ -0,0 +1,2 @@
+# Tell HBase whether it should manage it's own instance of Zookeeper or not.
```

Comment:
let's copy the whole `hbase-env.sh` file from https://github.com/RoySRose/pinpoint-docker/blob/SpringBoot/pinpoint-hbase/hbase-env.sh

even if rest of them are all commented since this file is generated by hbase.

Reply:
Okay I agree with that suggestion